scripts/pre-commit

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+echo "********************************"
+echo "Checking code formatting"
+echo "********************************"
+
+ktlint --android domain/src/**/*.kt utility/src/**/*.kt data/src/**/*.kt app/src/**/*.kt testing/src/**/*.kt
```

Comment:
Nit: Can we order them alphabetically i.e. app/data/domain/testing/utility

Reply:
Done